### ISSUE TITLE
Bug with villager's trades after deleting the mod

### ISSUE THREAD

**Issue by Habciak** [glisco03/numismatic-overhaul#154]
I deleted this mod on my server and now villagers are bugged. They have empty trades with "air". How can I fix that?

**Comment by Noaaan** [comment]
I do not think this is fixable. This will happen with any Villager who has a trade with a removed item from the game.
